src/db/00002_cleanup_indices.sql

```diff
@@ -0,0 +1,12 @@
+BEGIN;
+
+-- Remove automatically created indices 
+DROP INDEX IF EXISTS states_sort_key_index;
+
+DROP INDEX IF EXISTS black_list_contract_tx_id_index;
```

Comment:
why are you removing this index? and the one for the errors?

Reply:
Just a cleanup